crates/bws/CHANGELOG.md

```diff
@@ -7,13 +7,18 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.0.0] - 2024-09-26
+
 ### Added
 
 - The ability to edit unassigned secrets with direct permissions. (#906)
+- A way to opt out of state by default with the new state_opt_out key (#930)
```

Comment:
Should we link to any documentation of this?
```suggestion
- A way to opt out of state with the new `state_opt_out` setting (#930)
```

Reply:
Great idea. I've brought this up with the docs team, if we can get docs made for this I can update it!

Reply:
I think we should change the phrasing since `opt out of state by default` is vague, do we by default opt out of state? But we don't have to link it to our docs.

Reply:
No, we currently use state by default. `state_opt_out` allows you to opt out of using state files. We use state files by default to help prevent the rate limit errors the user would normally run into.

Reply:
I'm aware, but the phrasing of "A way to opt out of state by default" lacks clarity. Are we opting out of state by default?

If you want to keep default you can switch position to `- A way to opt out of the default state with the new `state_opt_out` setting (#930)`

Reply:
Ah, I see what you mean. Updated in https://github.com/bitwarden/sdk/pull/1122/commits/28090fcbde04ebb3ea8cd23520fd46055325b3e8, let me know what you think. Thanks!